WTWSMS/decisions/VIET_culture_decisions.txt

```diff
@@ -981,13 +979,12 @@ decisions = {
 			NOT = { religion_group = muslim }
 			OR = {
 				NOT = { religion_group = pagan_group }
-				religion = zun_pagan
-				religion = zun_pagan_reformed
 				AND = {
 					religion_group = pagan_group
-					realm_size = 80
+					realm_size = 20
```

Comment:
Anyone with money should be able to support the Silk Road, _**IF**_ they are located ON the Silk Road. I don't know how to code that, so I used this as a substitute.